docs/mkdocs.yml

```diff
@@ -34,33 +34,56 @@ site_dir: 'generated'
 nav:
   - Welcome:
       - Landing: index.md 
-  - Overview:
-      - Readme: readme.md
-      - User guide: direct/user-guide.md
+  - What is KubeStellar?:
+      - Overview: readme.md
+#      - User guide: direct/user-guide.md
       - Architecture: direct/architecture.md
+      - Related:
+        - KubeFlex: direct/kubeflex-intro.md
+        - KubeStellar Galaxy: direct/galaxy-intro.md
       - Release-notes: direct/release-notes.md
-  - Getting Started: 
-          - Pre-reqs: direct/pre-reqs.md
+  - Using Kubestellar:
+          - Basics: direct/usage-basics.md
+          - Prerequisites: direct/pre-reqs.md
+#              - Pre-reqs: direct/pre-reqs.md
           - Setting up KubeStellar:
-              - Using existing hosting cluster: direct/hosting-cluster.md
-              - KubeStellar on KIND: direct/ks-on-kind.md
-              - KubeStellar on K3D: direct/deploy-on-k3d.md
-              - KubeStellar on OCP: direct/ks-on-ocp.md
+              - Set Up Kubestellar Core Components: # common-subs/placeholder.md
+                - Try the "Common-setup": direct/common-setup-intro.md
+                - Step by Step: direct/common-setup-step-by-step.md
+                - (experimental) Via Helm Chart: direct/core-chart.md
+#                - Prepare the environment: common-subs/placeholder.md
+#                - Initialize Kubeflex: common-subs/placeholder.md
+#                - Install Kubestellar Core components: common-subs/placeholder.md
+#                - Create the Inventory & Transport Space: common-subs/placeholder.md
+#                - Create the Workload Description Space: common-subs/placeholder.md
+              - Set up and Register Workload Execution Clusters (WECs): direct/example-wecs.md
+#              - Register WECS with core: common-subs/placeholder.md
+              - Commission Workloads: direct/commisssion-workloads.md
+#          - Modifying Workloads: common-subs/placeholder.md
+          - Installation & Usage Examples:
```

Comment:
Why is "Installation" here, separate from the previous "Set Up" section? They seem roughly synonymous.

Reply:
I don't  think any of these steps should be in the mkdocs menu .   I think we want a very simple menu:

- Using Kubestellar:
          - Setup KubeStellar:  "setup-ks.md"  (this should include a pointer to "pre-req")
          - Deploy a workflow to multiple clusters:  "deploy-example.md"
  
 Everything else (if at all needed) should be just linked from within those docs.

Reply:
My point here is not about where the tree is clipped for menu inclusion, my point is about the same sort of information not collected in one branch of the tree.

I can see an argument for keeping the discussion of pre-reqs as a peer of installation rather than a part of installation. Someone evaluating or planning will need to know the pre-reqs but not particularly care about the actual installation steps.

Reply:
The tree was expanded based on my *excruciating* experience bouncing back and forth between files linked in the old 'common setup" instructions. If there are multiple files with instructions that have to be done in order, those should be in the left menu tree so they are always visible *in order*. 

I had not come to a resolution of how to organize the installation instructions which were part of the examples section vs  those in a getting started section (since at the moment they are versions of the same process).